test/e2e/e2e.go

```diff
@@ -317,16 +317,16 @@ func setupSuite() {
 
 // logClusterImageSources writes out cluster image sources.
 func logClusterImageSources() {
-	masterImg, nodeImg, err := lookupClusterImageSources()
+	controlPlaneImg, nodeImg, err := lookupClusterImageSources()
 	if err != nil {
 		framework.Logf("Cluster image sources lookup failed: %v\n", err)
 		return
 	}
-	framework.Logf("cluster-master-image: %s", masterImg)
+	framework.Logf("cluster-control-plane-image: %s", controlPlaneImg)
 	framework.Logf("cluster-node-image: %s", nodeImg)
 
 	images := map[string]string{
-		"master_os_image": masterImg,
+		"control_plane_os_image": contorlPlaneImg,
```

Comment:
maybe:
`control_plane_node_os_image`
_and_
`data_plane_node_os_image`
So we can specify the type of node that the os image belongs to instead of just saying `node_os_image`. Also, typo in kv pair for "control_plane_os_image": `controlPlaneImg`.

So maybe we can do something like this:
```go
images := map[string]string{
    "control_plane_os_image": controlPlaneImg,
    "data_plane_os_image": dataPlaneNodeImg,
}

Reply:
also typo in `contorlPlaneImg`